test/Test/Hspec/Expectations/PrettySpec.hs

```diff
@@ -113,4 +119,4 @@ spec = do
       (return () `shouldThrow` anyErrorCall) `shouldThrow` expectationFailed "did not get expected exception: ErrorCall"
 
     it "fails, if a required specific exception is not thrown" $ do
-      (error "foo" `shouldThrow` errorCall "foobar") `shouldThrow` expectationFailed "predicate failed on expected exception: ErrorCall (foo)"
+      (error "foo" `shouldThrow` errorCall "foobar") `shouldThrow` expectationFailedPrefix "predicate failed on expected exception: ErrorCall (foo"
```

Comment:
A bit of context: it seems the error messages now include the stack trace as well, so the error message in this test is no longer just "predicate failed on expected exception: ErrorCall (foo)", it's actually:

```
       predicate failed on expected exception: ErrorCall (foo
       CallStack (from HasCallStack):
         error, called at /Users/dcastro/Dropbox/Projects/Haskell/test-project/test/Test/TestSpec.hs:22:7 in main:Test.TestSpec)
```

So I figured testing for a prefix of the error message would be enough.